promgen/views.py

```diff
@@ -626,6 +615,10 @@ def post(self, request, pk):
 
 class FarmLink(LoginRequiredMixin, View):
     def get(self, request, pk, source):
+        if source == discovery.FARM_DEFAULT:
+            messages.error(request, "Cannot link to local farm")
+            return HttpResponseRedirect(reverse("project-detail", args=[pk]) + "#hosts")
+
```

Comment:
In theory this should never happen, but is good to have this check in case someone tries to hack the system :+1: